pyriemann_qiskit/classification.py

```diff
@@ -0,0 +1,296 @@
+"""Module for classification function."""
+import numpy as np
+
+from sklearn.base import BaseEstimator, ClassifierMixin
+from sklearn.model_selection import train_test_split
+
+from qiskit import BasicAer, IBMQ
+from qiskit.circuit.library import ZZFeatureMap, TwoLocal
+from qiskit.aqua import QuantumInstance, aqua_globals
+from qiskit.aqua.quantum_instance import logger
+from qiskit.aqua.algorithms import QSVM, SklearnSVM, VQC
+from qiskit.aqua.utils import get_feature_dimension
+from qiskit.providers.ibmq import least_busy
+from qiskit.aqua.components.optimizers import SPSA
+from datetime import datetime
+import logging
+logger.level = logging.INFO
+
+
+class QuanticBase(BaseEstimator, ClassifierMixin):
```

Comment:
`QuanticBase` seems a little bit too generic name: for eg, what would be the name of a base class for quantic regression?
`QuanticBase` could be renamed `QuanticClassifierBase`, avoiding future conflicts when adding new classes, like `QuanticRegressorBase`.